cmd/fund/fund.go

```diff
@@ -0,0 +1,132 @@
+package fund
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"log"
+	"math/big"
+
+	_ "embed"
+
+	"github.com/chenzhijie/go-web3"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+var (
+	//go:embed usage.md
+	usage                string
+	walletCount          int
+	fundingWalletAddress string
+	fundingWalletPK      string
+	chainID              int
+	chainRPC             string
+	walletFundingAmt     float64
+	walletFundingGas     uint64
+)
+
+func generateWalletAddresses(numWallets int) ([]*common.Address, error) {
+	var addresses []*common.Address
+
+	for i := 0; i < numWallets; i++ {
+		account, err := crypto.GenerateKey()
+		if err != nil {
+			log.Fatal(err)
+		}
+		addr := crypto.PubkeyToAddress(account.PublicKey)
+		addresses = append(addresses, &addr)
```

Comment:
nit: change related to switching to pre-allocated slice

```suggestion
		addresses[i] = &addr
```